tests/fixtures/remove_class/assertEqual_in.py

```diff
@@ -6,4 +6,22 @@ def test_you(self):
 class TestAssertNotEqual(TestCase):
 
     def test_you(self):
-        self.assertNotEqual(abc, 'xxx')
+        """
+        hello
+        """
+        pass
+        if True:
+            if False:
+                pass
```

Comment:
Why this doc-string and this code? Is it to test dedenting? So please add a brief comment above.